presto-spi/src/main/java/com/facebook/presto/spi/connector/ConnectorMetadata.java

```diff
@@ -354,4 +356,24 @@ default void revokeTablePrivileges(ConnectorSession session, SchemaTableName tab
     {
         throw new PrestoException(NOT_SUPPORTED, "This connector does not support revokes");
     }
+
+    default TableIdentity getTableIdentity(ConnectorTableHandle connectorTableHandle)
```

Comment:
I'd order these `getTableIdentity`, `deserializeTableIdentity`, `getColumnIdentity` and then `deserializeColumnIdentity`